hazelcast/src/main/java/com/hazelcast/map/impl/SimpleEntryView.java

```diff
@@ -240,6 +255,7 @@ public void writeData(ObjectDataOutput out) throws IOException {
         // writes the deprecated evictionCriteriaNumber to the data output (client protocol compatibility)
         out.writeLong(0);
         out.writeLong(ttl);
+        out.writeLong(maxIdle);
```

Comment:
RU compatibility with 3.10? Seems should check cluster-version like [this](https://github.com/hazelcast/hazelcast/blob/9ae4484b402aa769ec06410a147848c016d2bd7f/hazelcast/src/main/java/com/hazelcast/config/MapConfig.java#L1104)